apps/heft/src/plugins/CopyFilesPlugin.ts

```diff
@@ -160,35 +160,19 @@ export class CopyFilesPlugin implements IHeftPlugin {
       Constants.maxParallelism,
       async (copyDescriptor: ICopyFileDescriptor) => {
         if (copyDescriptor.hardlink) {
-          const hardlinkPromises: Promise<void>[] = copyDescriptor.destinationFilePaths.map(
-            (destinationFilePath) => {
-              return FileSystem.createHardLinkAsync({
-                linkTargetPath: copyDescriptor.sourceFilePath,
-                newLinkPath: destinationFilePath,
-                alreadyExistsBehavior: AlreadyExistsBehavior.Overwrite
-              });
-            }
-          );
-          await Promise.all(hardlinkPromises);
-
           linkedFileCount++;
+          return FileSystem.createHardLinkAsync({
```

Comment:
```suggestion
          await FileSystem.createHardLinkAsync({
```

Otherwise, the stacktrace on an error won't show this line.